components/board/client.go

```diff
@@ -75,6 +76,14 @@ func (c *client) AnalogReaderByName(name string) (AnalogReader, bool) {
 	}, true
 }
 
+func (c *client) AnalogWriterByName(name string) (AnalogWriter, bool) {
+	return &analogWriterClient{
+		client:    c,
+		boardName: c.info.name,
+		pinName:   name,
+	}, true
```

Comment:
...this file is basically the Go SDK, which runs client-side and sends GRPCs to the server to get stuff done, right? I had thought the boolean returned would indicate whether we've got a valid writer, but this makes it look like it's always valid, even if the server is running on a board with no writers, even if the named pin doesn't exist. Any chance we can do something smarter here? and again a valid response is "this is consistent with the rest of the implementation. If you want to do something smarter, change all the implementations in a separate PR."